### PR TITLE
Document the formatting capabilities of the table control

### LINKED ISSUE #520: Specify what is number_format for Tables
**Description**
No documentation is provided to know what should be put in number format.

**Expected change**
Some documentation or link should be placed to help the user understand what can be put in number_format.

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Resolves #520.

This needs to be complemented by an example in the taipy repository (`doc/examples/controls/table-formatting.py`).

### CODE REVIEW

File: docs/manuals/gui/viselements/table.md_template [hunk 1]
```diff
@@ -36,6 +36,202 @@ The table component supports three display modes:
...
+<figure class="tp-center">
+    <img src="../table-format3-d.png" class="visible-dark" />
+    <img src="../table-format3-l.png" class="visible-light"/>
+    <figcaption>Formatting the numerical values</figcaption>
+</figure>
+
+Although we only have two decimal places for stock prices, which was the point of setting the
+[*number_format*](#p-number_format) property, the format has impacted all numerical values,
+including the values of the *volume* column, which is not what we want: volumes are integer values,
+they should have to decimal places at all.
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/table.md_template]: to => no ?

**Review by FredLL-Avaiga — APPROVED** [review]
1 small typo

File: docs/manuals/gui/viselements/table.md_template (hunk 1 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/table.md_template]: Indeed!

**Review by FlorianJacta — APPROVED** [review]
I like this *column* property, and the doc is really great!

**Review by namnguyen20999 — APPROVED** [review]
Looks great !!!

**Pull request merged**